data/src/main/kotlin/com/vultisig/wallet/data/blockchain/cosmos/staking/CosmosStakingSignDataResolver.kt

```
@@ -0,0 +1,235 @@
+            preflight(validator, chain)
+            CosmosStakingHelper.encodeWithdrawDelegatorReward(
+                delegator = delegator,
+                validator = validator,
+            )
+        }
+    }
+
+    private fun preflight(validator: String, chain: Chain) {
+        try {
+            ValidatorBech32Preflight.validate(validator, chain)
+        } catch (e: ValidatorBech32Preflight.ValidatorBech32Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)